docs/Automation.md

```diff
@@ -0,0 +1,261 @@
+            /// </summary>
+            static void Main(string[] args)
+            {
+                var parameters = new Dictionary<string, string>();
+                string secondaryConfigFile = string.Empty;
+                char[] delimiters = {'='};
+                foreach (string arg in args)
+                {
+                    string[] pieces = arg.Split(delimiters);
+                    if (pieces.Length == 2)
+                    {
+                        string key = pieces[0].Trim();
+                        string value = pieces[1].Trim();
+                        if (!string.IsNullOrWhiteSpace(key) && !string.IsNullOrWhiteSpace(value))
+                        {
+                            // Special case for SecondaryConfigFile
+                            if (key.Equals("SecondaryConfigFile", StringComparison.OrdinalIgnoreCase))
+                            {
+                                secondaryConfigFile = value;
+                            }
+                            else
+                            {
+                                parameters[key] = value;
+                            }
+                            continue;
+                        }
+                    }
+                    Console.WriteLine("Ignoring malformed input: {0}", arg);
+                };
+                Console.WriteLine(StartCommand.Execute(parameters secondaryConfigFile).ToString());
+                int autoFileId = 0;
+                while (true)
+                {
+                    Console.Write("Enter process ID to capture (blank to exit): ");
+                    string input = Console.ReadLine();
+                    if (input == string.Empty)
+                        break;
+                    if (!int.TryParse(input, out int processId))
+                    {
+                        Console.WriteLine("Not a valid int: " + input);
+                        continue;
+                    }
+                    Dictionary<string, string> snapshotParameters = new Dictionary<string, string>
+                    {
+                        { CommandConstStrings.TargetProcessId, input },
+                        { CommandConstStrings.OutputFile, autoFileId++.ToString() },
+                    };
+                    Console.WriteLine(SnapshotCommand.Execute(snapshotParameters).ToString());
+                }
+                Console.WriteLine(StopCommand.Execute().ToString());
+            }
+        }
+    }
+```
+
+#### From PowerShell
+-   Start your PowerShell script from the folder where the AxeWindows
+    DLLs are located.
+-   Load the assembly via **Import-Module Axe.Windows.Automation.dll**.
+-   Access the Start command via the **Start-AxeWindows** cmdlet.
+-   Access the Snapshot command via the **Invoke-Snapshot** cmdlet.
+-   Access the Stop command via the **Stop-AxeWindows** cmdlet.
+
+Sample PowerShell script (the 2 second delay exists to give the app time to initialize before it is scanned):
```

Comment:
This script will work only if the instance of notepad.exe that we start is the only instance in the entire system. That's a reasonable limitation for a sample script, although we probably ought to document it as such